src/dialogs/examples/basic/AdaptiveCardDialog.ts

```diff
@@ -0,0 +1,451 @@
+                                                    "color": null,
+                                                    "isSubtle": true,
+                                                    "text": "Hello!",
+                                                    "horizontalAlignment": "left",
+                                                    "wrap": false,
+                                                    "maxLines": 0,
+                                                    "separation": null,
+                                                },
+                                                {
+                                                    "type": "TextBlock",
+                                                    "size": null,
+                                                    "weight": null,
+                                                    "color": null,
+                                                    "isSubtle": false,
+                                                    "text": "Are you looking for a Tab or Bot?",
+                                                    "horizontalAlignment": "left",
+                                                    "wrap": true,
+                                                    "maxLines": 0,
+                                                    "separation": null,
+                                                },
+                                            ],
+                                            "style": null,
+                                            "separation": null,
+                                        },
+                                    ],
+                                },
+                                // input toggle collects a true/false response from the user
+                                {
+                                    "type": "Input.Toggle",
+                                    "title": "I accept the terms and conditions (True/False)",
+                                    "valueOn": "true", // the value when toggle is on (default: true)
+                                    "valueOff": "false", // the value when toggle is off (default: false)
+                                    "id": "AcceptsTerms",
+                                    "isRequired": false,
+                                    "separation": null,
+                                },
+                            ],
+                        },
+                    ],
+                    "actions": [
+                        // submit action gathers up input fields, merges with optional data field and generates event to client asking for data to be submitted
+                        {
+                            "type": "Action.Submit",
+                            "title": "Submit",
+                            "speak": "<s>Search</s>",
+                            "data": {
+                                "dialog": "AdaptiveCardDialog",
```

Comment:
Remove the 'dialog' property and add isFromAdaptiveCard: true. Next, add a 'messageText' property that is populated with a constant in this dialog - AdaptiveCardDialog.textToTriggerThisDialog = "Adaptive Card";

Reply:
Done.